src/action.ts

```diff
@@ -57,6 +57,9 @@ export default async function main() {
     ? appendToPreReleaseTag
     : currentBranch;
 
+// Sanitize identifier according to https://semver.org/#backusnaur-form-grammar-for-valid-semver-versions
+let sanitized = identifier.replace(/[^a-zA-Z0-9-]/g, '-');
```

Comment:
Can you please also fix the indentation? Thanks! :)

Reply:
Fixed. :)